lane.py

```diff
@@ -2122,7 +2104,7 @@ def _featured_works_with_lanes(
 
         # Ask the search engine for works from every lane we're given.
         for lane in lanes:
-            for work in lane.works_from_search_index(
+            for work in lane.works(
```

Comment:
Same problem here, but in reverse -- this is being called for lanes that get works from the database, not just lanes that get works from the search index. The lane implementation controls which is used in a specific case.